### PR TITLE
picture cards: add person image support

### PR DESCRIPTION
**Pull request by Quentame**
## Proposed change

Add doc for https://github.com/home-assistant/frontend/pull/20593

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/frontend/pull/20593
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **New Features**
	- Introduced an optional `image_entity` field across various dashboard configurations, allowing users to specify an image or person entity for display. This enhances visual representation and flexibility.
- **Documentation**
	- Clarified the description of the `entity_id` field, specifying that it can refer to a camera, image, or person, improving user understanding of its context.

### CODE REVIEW

File: source/_dashboards/picture-entity.markdown [hunk 1]
```diff
@@ -32,7 +32,7 @@ type:
   type: string
 entity:
   required: true
-  description: "An `entity_id` used for the picture."
+  description: "A camera, image or person `entity_id` used for the picture."
```
Comment (c0ffeeca7) [inline source/_dashboards/picture-entity.markdown]:
```suggestion
  description: "A camera, image, or person `entity_id` used for the picture."
```

**Review by c0ffeeca7 — CHANGES_REQUESTED** [review]
Tiny tweak

**Review by c0ffeeca7 — DISMISSED** [review]
Thank you, @Quentame :+1:

**Review by silamon — COMMENTED** [review]
During code review of the frontend pr, it became clear that image_entity looks for an entity to use the image from in both the picture elements card and picture glance card. Therefore person_entity and image_entity have been combined into image_entity to prevent having a separate option for each supported entity type. Image_entity can be either an image or a person entity.

**Review by silamon — APPROVED** [review]

**Review by c0ffeeca7 — APPROVED** [review]

**Pull request merged**